bedrock/firefox/templates/firefox/new/onboarding/scene1.html

```diff
@@ -117,10 +131,20 @@ <h2 class="forward">{{_('Go Forward') }}</h2>
 
   <div id="other-platforms">
     <div class="container">
-      <section class="section-other-platforms">
-        <h4>{{_('Firefox for other platforms') }}</h4>
+      {#
+        when l10n tag 'firefox_new_install_options' is removed, remove
+        'no-lang-link' class below and from the CSS file
+      #}
+      <section class="section-other-platforms{% if l10n_has_tag('firefox_new_install_options') %} no-lang-link{% endif %}">
+        <h4>
+        {% if l10n_has_tag('firefox_new_install_options') %}
+          {{ _('Advanced Install Options & Other Platforms') }}
```

Comment:
Ideally, yes. For example: what if you want to use a different case for the two contexts (e.g. all lowercase in the button)?